WebAPI/Controllers/ProductsController.cs

```diff
@@ -32,25 +41,47 @@ public async Task<ActionResult<Task>> Add(ProductDetailDto product)
         return BadRequest(taskResult);
     }
 
+    /// <summary>
+    /// Updates a specific Product.
+    /// </summary>
+    /// <param name="product"></param>
+    /// <returns>Updated Product</returns>
     [HttpPut]
     public async Task<ActionResult<ProductDetailDto>> Update(ProductDetailDto product)
     {
         var result = await _productService.UpdateAsync(product);
 
-        if(result != null)
+        if (result != null)
         {
             return result;
         }
- 
+
         return BadRequest(result);
     }
 
-    [HttpDelete("{id}:guid")]
+    /// <summary>
+    /// Removes a specific Product.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns>Task Result</returns>
+    /// <remarks>
+    /// Sample request:
+    ///
+    ///     Delete /Products
+    ///     {
+    ///        "Id": "031ec27b-f9e4-428f-9a42-292080fe9954"
+    ///     }
+    /// </remarks>
```

Comment:
Let's add the same style for the rest of endpoints to make doc consistent and clear, please :)